test_unstructured_ingest/metrics/all-docs-cct.tsv

```diff
@@ -1,4 +1,4 @@
 filename	connector	cct-accuracy	cct-%missing
 handbook-1p.docx	box	0.974	0.03
 example-10k.html	local	0.686	0.04
-IRS-form-1987.pdf	azure	0.783	0.13
+science-exploration-1p.pptx	dropbox	0.861	0.09

```

Comment:
why is dropbox reported here?

Reply:
That's probably the version lingered in my local. Let me try to resolve that with the new src/dest setting.